projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -458,6 +462,67 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 	}
 }
 
+/**
+ * Force a variation (control/treatment) for the Remove Duplicate Views experiment.
+ *
+ * @return void
+ */
+function wpcom_force_assign_variation_for_remove_duplicate_views_experiment() {
+	if ( ! isset( $_GET['force-assign-rdv-variation'] ) ) {
+		return;
+	}
+
+	$assignment = in_array( $_GET['force-assign-rdv-variation'], array( 'control', 'treatment' ), true ) ? sanitize_text_field( wp_unslash( $_GET['force-assign-rdv-variation'] ) ) : false;
+
+	if ( ! $assignment ) {
+		return;
+	}
+
+	/**
+	 * Setting the option globally (third parameter) will have the following behavior:
+	 * - On Simple Sites, the option will be shared between them.
+	 * - On Atomic Sites, the option will NOT be shared.
+	 *
+	 * This also means that if a user has a Simple Sites and Atomic sites, the option will not be shared between them.
+	 *
+	 * For example, if the option is set on a Simple Site, every other site will get it, except for the Atomic ones.
+	 * If the option is set on an Atomic Site, this will apply only on this site - it won't be shared between the other Atomic Sites OR Simple Sites.
+	 *
+	 * Since this should be used only in exceptional cases, there's no need to implement something better.
+	 */
+	update_user_option( get_current_user_id(), RDV_EXPERIMENT_FORCE_ASSIGN_OPTION, $assignment, true );
+}
+
+/**
+ * Reset the assignment cache for the Remove duplicate views experiment.
+ *
+ * @return void
+ */
+function wpcom_reset_assignment_for_remove_duplicate_views_experiment() {
+	if ( ! isset( $_GET['force-reset-rdv-variation'] ) ) {
+		return;
+	}
+
+	/**
+	 * Setting the option globally (third parameter) will have the following behavior:
+	 * - On Simple Sites, the option will be shared between them.
+	 * - On Atomic Sites, the option will NOT be shared.
+	 *
+	 * This also means that if a user has a Simple Sites and Atomic sites, the option will not be shared between them.
+	 *
+	 * For example, if the option is set on a Simple Site, every other site will get it, except for the Atomic ones.
+	 * If the option is set on an Atomic Site, this will apply only on this site - it won't be shared between the other Atomic Sites OR Simple Sites.
```

Comment:
Can you also mention that if a site is simple and goes atomic then it won't go across

Reply:
Sure!